Added support for Roland mixers V-8HD and V-60HD

JWandscheer opened the pull request:
Added Support for Roland V-8HD and V-60HD.
The V-8HD needs to be connected directly to the hub via USB, the connection is based on midi. See
#51 .

The tally status of Roland V-60HD is requested by its ip address.

czenker reviewed (commented):
Great job, @JWandscheer!

You did very well and found all the places that needed to have changes (happy easter by the way :rabbit:). Unfortunately I do not have access to any of the hardware, but the code makes a very good impression and it was obviously quite some work. So I'm going to merge this.

I have added a few comments, but they are non-blocking. Also, if it is not too hard a few functional tests would be great.

A test against the real hardware (as in [manual_atem.spec.ts](https://github.com/wifi-tally/wifi-tally/blob/master/hub/cypress/integration/manual_atem.spec.ts)) could help to ensure the connector still works next year after a few updates from Roland.

A functional test for the connector with mocked API calls (as [VmixConnector.spec.js](https://github.com/wifi-tally/wifi-tally/blob/master/hub/src/mixer/vmix/VmixConnector.spec.js) will make refactorings easier. If `axios` allows easy mocking of HTTP calls for tests, this might even convince me to leave the library in. :laughing: . Mocking `midi` might be harder, but if it is easy there should be a test.

JWandscheer commented:
Thanks for yout feedback @czenker! Will take a look at this next week and improve some things - will do my best with mocking and decide weather to remove axios or leave it in :-)

JWandscheer commented:
Just implemented your feedback @czenker , did my very best in testing^^

I haven't written a test on real hardware (yet), because I don't have access to the mixer - trying to do that later, but it can take some time.
I added a functional test for the V-60HD, not yet for the V-8HD, as I am not sure how to handle midi-mocking. Maybe [midi-test](https://www.npmjs.com/package/midi-test) might be helpful here?

Since I don't have the V-60HD here, the code would have to be tested again on real hardware anyway, maybe @Brunsben can do that?

Brunsben commented:
> Since I don't have the V-8HD here, the code would have to be tested again on real hardware anyway, maybe @Brunsben can do that?

Sure I can.
I'll get in touch with you, then we can coordinate that once.
